### PR TITLE
admin menu

### PR DESCRIPTION
**Pull request by eliatib**

### CODE REVIEW

**Comment by ondrej-kuhnel** [comment]
@eliatib Where is the menu?

**Comment by eliatib** [comment]
> @eliatib Where is the menu?

admin menu -> customer -> messages

File: src/Controller/ContactFormController.php [hunk 1]
```diff
@@ -64,6 +63,15 @@ public function __construct(
         $this->channelContext = $channelContext;
     }
 
+    public function showMessageAction(int $id)
+    {
+        $contactMessages = $this->entityManager->getRepository(ContactFormMessage::class)->find($id);
```
Comment (ondrej-kuhnel) [inline src/Controller/ContactFormController.php]: You should use `ContactFormRepository` injected in Controller

File: src/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -7,6 +7,24 @@ mango_sylius:
         message: 'Message'
         success: 'The message was sent with success'
         error: 'An error occurred when sending the message'
+        time: 'Send at %Time%'
```
Comment (ondrej-kuhnel) [inline src/Resources/translations/messages.en.yml]: Always use a lowercase letter in variables

File: src/Resources/views/ContactForm/ContactFormShopManagerEmail.html.twig [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% block subject %}
-    {{ 'mango_sylius.contactForm.subject.shopManager'|trans({'%customerName%': contact.senderName}) }}
+    {{ 'mango_sylius.contactForm.subject.shopManager'|trans({'%CustomerName%': contact.senderName}) }}
```
Comment (ondrej-kuhnel) [inline src/Resources/views/ContactForm/ContactFormShopManagerEmail.html.twig]: Always use a lowercase letter in variables

File: src/Resources/views/ContactForm/show.html.twig [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{% extends '@SyliusAdmin/layout.html.twig' %}
+
+{% block title %}{{ 'mango_sylius.contactForm.ui.detailTitle'|trans ({'%Contact%': message.senderName|first })}}{% endblock %}
+
+{% block content %}
+
+
+        <div class="ui segment">
+            {{ 'mango_sylius.contactForm.name'|trans ({'%Name%' : message.senderName})}}
```
Comment (ondrej-kuhnel) [inline src/Resources/views/ContactForm/show.html.twig]: Always use a lowercase letter in variables

File: src/Resources/views/ContactForm/show.html.twig [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+            </br></br>
+            {{ message.message }}
+            </br></br>
+            {{ 'mango_sylius.contactForm.time'|trans ({'%Time%': message.sendTime|date('Y-m-d H:i:s') })}}
```
Comment (ondrej-kuhnel) [inline src/Resources/views/ContactForm/show.html.twig]: Always use a lowercase letter in variables

File: src/Resources/translations/messages.en.yml [hunk 6]
```diff
@@ -7,6 +7,24 @@ mango_sylius:
...
+        name: 'From %Name%'
         subject:
-            shopManager: 'Message From %customerName%'
-            customer: 'Message sent'
\ No newline at end of file
+            shopManager: 'Message From %CustomerName%'
```
Comment (ondrej-kuhnel) [inline src/Resources/translations/messages.en.yml]: Always use a lowercase letter in variables

File: src/Resources/translations/messages.en.yml [hunk 7]
```diff
@@ -7,6 +7,24 @@ mango_sylius:
...
+            customer: 'Message sent'
+        ui:
+            title: 'Messages'
+            subtitle: 'Manage your messages'
+            detailTitle: 'Messages from %Contact%'
```
Comment (ondrej-kuhnel) [inline src/Resources/translations/messages.en.yml]: Always use a lowercase letter in variables

File: src/Resources/translations/messages.en.yml [hunk 8]
```diff
@@ -7,6 +7,24 @@ mango_sylius:
         message: 'Message'
         success: 'The message was sent with success'
         error: 'An error occurred when sending the message'
+        time: 'Send at %Time%'
+        name: 'From %Name%'
```
Comment (ondrej-kuhnel) [inline src/Resources/translations/messages.en.yml]: Always use a lowercase letter in variables

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: src/Resources/translations/messages.en.yml (hunk 8 above)
Comment (eliatib) [reply src/Resources/translations/messages.en.yml]: DONE

File: src/Resources/translations/messages.en.yml (hunk 7 above)
Comment (eliatib) [reply src/Resources/translations/messages.en.yml]: DONE

File: src/Resources/translations/messages.en.yml (hunk 6 above)
Comment (eliatib) [reply src/Resources/translations/messages.en.yml]: DONE

File: src/Resources/views/ContactForm/show.html.twig (hunk 5 above)
Comment (eliatib) [reply src/Resources/views/ContactForm/show.html.twig]: DONE

File: src/Resources/views/ContactForm/show.html.twig (hunk 4 above)
Comment (eliatib) [reply src/Resources/views/ContactForm/show.html.twig]: DONE

File: src/Resources/views/ContactForm/ContactFormShopManagerEmail.html.twig (hunk 3 above)
Comment (eliatib) [reply src/Resources/views/ContactForm/ContactFormShopManagerEmail.html.twig]: DONE

File: src/Resources/translations/messages.en.yml (hunk 2 above)
Comment (eliatib) [reply src/Resources/translations/messages.en.yml]: DONE

File: src/Controller/ContactFormController.php (hunk 1 above)
Comment (eliatib) [reply src/Controller/ContactFormController.php]: DONE

**Review by ondrej-kuhnel — APPROVED** [review]

**Pull request merged**